content/developers/developer-patterns/scitt-api/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: "Quickstart: SCITT for Supply Chains (Preview)"
+description: "Getting Started with SCITT, enabling Software Supply Chain scenarios (Preview)"
+lead: "Push a collection of Supply Chain Statements using SCITT APIs"
+date: 2021-06-09T13:49:35+01:00
+lastmod: 2021-06-09T13:49:35+01:00
+draft: false
+images: []
+menu:
+  developers:
+    parent: "developer-patterns"
+weight: 112
+toc: true
+aliases: 
+  - /docs/developer-patterns/scitt-api/
+---
+
+{{< caution >}}
+The SCITT API is currently in preview and subject to change
+{{< /caution >}}
+
+The **S**upply **C**hain **I**ntegrity, **T**ransparency and **T**rust (SCITT) initiative is a set of [IETF standards](https://datatracker.ietf.org/group/scitt/documents/) for managing the compliance and auditability of goods and services across end-to-end supply chains.
+SCITT supports the ongoing verification of goods and services where the authenticity of entities, evidence, policy, and artifacts can be assured and the actions of entities can be guaranteed to be authorized, non-repudiable, immutable, and auditable.
+
+To assure insights to supply chain artifacts are current, the SCITT APIs provide a correlation of statements for a specific artifact, allowing newer information to be registered for the most up to date insights.
+
+This quickstart will:
+
+1. create, or use an existing a key to sign a collection of statements about an artifact
+1. create and register an SBOM for the artifact
+1. create and register an attestation for the artifact
+1. query a collection of statements about the artifact
+
+## Prerequisites
+
+- [A DataTrails subscription](https://app.datatrails.ai/signup)
+- [DataTrails sample code](#datatrails-sample-code)
```

Comment:
minor: we should put the sample repo here now instead of a placeholder?

Reply:
Yeah, it's a link to the section for cloning the repo. If you can view the page in [preview mode](https://docs.datatrails.ai/contributing/getting-started/pulling-and-building-rkvst-docs/), it jumps to scitt-api/#datatrails-sample-code heading